.github/workflows/build.yml

```diff
@@ -24,6 +24,7 @@ jobs:
   build:
     continue-on-error: ${{ endsWith(inputs.python-version, '-dev') || contains(fromJSON('["3.7", "pypy3.7"]'), inputs.python-version) || inputs.rust == 'beta' || inputs.rust == 'nightly' }}
     runs-on: ${{ inputs.os }}
+    if: ${{ !(startsWith(inputs.python-version, 'graalpy') && startsWith(inputs.os, 'windows')) }}
```

Comment:
Is Windows support now available, or is this left for a follow-up?

Reply:
It is available, but I would leave it as a follow up since this is our first release with C API support on Windows and I haven't managed to set up a dev environment with PyO3 on Windows so far :)

Reply:
Yep makes sense 👍